### PR TITLE
Primary Changes: Job Seeker's Application Job Details | Minor: postJObs

### PR DESCRIPTION
**Pull request by calvincchong**
Added some pics folx
- Updating the notes doesn't make a post to backend 
- comments currently aren't pulled from backend 
- notes styling doesn' stay the same 
- images are chosen at random. 

Nav Bar: 
- postJobs only for companies
- postJobs link added to the topbar
- upload reume link only for job seekers
- logout on settings menu only available if you are signed in. 

still got some bugs where on refresh sometimes it doesn't realize that the user is already logged in, particularly when we switch to the jobboard page. 

Screenshot 2023-01-19 at 9 47 57 PM
Screenshot 2023-01-19 at 9 47 49 PM
Screenshot 2023-01-19 at 9 47 43 PM

### CODE REVIEW

File: src/JobSeekerJobListCard.jsx [hunk 1]
```diff
@@ -15,9 +15,11 @@ const JobSeekerJobListCard = ({listing, seeDetailsVisibility}) => {
   const closeDate = listing.Job.close_date.slice(0, 10);
   const job_id = listing.job_id;
 
+  let jobDetails = {};
```
Comment (csb91) [inline src/JobSeekerJobListCard.jsx]: Might want to change this to state variable like company how company name is

File: src/JobSeekerJobListCard.jsx [hunk 2]
```diff
@@ -26,6 +28,7 @@ const JobSeekerJobListCard = ({listing, seeDetailsVisibility}) => {
       await axios.get(`http://localhost:3001/job?job_id=${job_id}`)
       .then(res => {
         console.log(res.data)
+        jobDetails = res.data;
```
Comment (csb91) [inline src/JobSeekerJobListCard.jsx]: ties in with comment above

File: src/JobSeekerJobListCard.jsx (hunk 1 above)
Comment (calvincchong) [reply src/JobSeekerJobListCard.jsx]: Updatedd

File: src/JobSeekerJobListCard.jsx (hunk 2 above)
Comment (calvincchong) [reply src/JobSeekerJobListCard.jsx]: updated!

**Review by csb91 — APPROVED** [review]

**Pull request merged**